webpack/scenes/Subscriptions/SubscriptionReducer.js

```diff
@@ -14,11 +14,7 @@ import {
   SUBSCRIPTIONS_QUANTITIES_FAILURE,
   SUBSCRIPTIONS_COLUMNS_REQUEST,
   UPDATE_SUBSCRIPTION_COLUMNS,
-  UPDATE_QUANTITY_REQUEST,
-  UPDATE_QUANTITY_SUCCESS,
-  UPDATE_QUANTITY_FAILURE,
```

Comment:
Mentioned off-thread, but commenting here for visibility: These are still dispatched, should this be handled differently? Right now, it will fall to the default state, suggesting no dispatch was needed.